### DISCUSSION TITLE
Renamer without scraping / using just sqlite DBs?

### DISCUSSION THREAD

**Discussion by verabox** [UltimaHoarder/UltimaScraper#1402] [Q&A] [answered]
Looked through as many of the discussions and recent threads and the helper script and I don't see any obvious way of doing this. I can certainly use the existing code to script something out if there isn't a way but before I spend my time on that I wanted to make sure I'm not missing something obvious. Script worked flawlessly for all of my old downloads of models I'm still subbed to/with scraping on. Just can't get it to work with older downloads that I'm subbed to and are still in their OG folder.

**Comment by UltimaHoarder** [comment, marked as answer]
It's something that I'll be including in the next version. As of now you can turn off the downloader and just have renaming.

**Reply by verabox** [reply]
Thanks! You'll save me the trouble of having to do this lol

**Reply by verabox** [reply]
FWIW I already tried to turn off the downloader -- it moved and renamed all of the active subs I had no problem, but it did not do it to my expired subs. I'll wait for your next update and just use that. Thanks for all of your work!
